src/reanimated2/animation/timing.ts

```diff
@@ -65,6 +65,29 @@ export const withTiming = function (
 ): Animation<TimingAnimation> {
   'worklet';
 
+  if (__DEV__) {
+    // validate config
+    if (userConfig?.easing) {
```

Comment:
Can we merge it into one condition (`__DEV__ && userConfig?.easing`) and save one level of indentation?